dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/rpc/remote/NettyClient.java

```diff
@@ -211,7 +211,8 @@ public RpcResponse sendMsg(Host host, RpcProtocol<RpcRequest> protocol, Boolean
         if (Boolean.TRUE.equals(async)) {
             result = new RpcResponse();
             result.setStatus((byte) 0);
-            result.setResult(true);
+            //firstly return a null value when meet the async call
+            result.setResult(null);
```

Comment:
It seems not the general processing if others need to judge `result==true`

Reply:
in my opinion , the return type of meothd in async call cannot used directly,it must use the callback mechanism to get the real response.
so the `result==true` can't be judge in the very fisrt time to use.
It's very awesome you can reply in this Spring Festival , thx